### ISSUE TITLE
Unable to open a wasm file (Mint22)

### ISSUE THREAD

**Issue by not-magic** [Yeicor/sdf-viewer#370]
So the instructions in the readme are pretty vague so maybe I am doing something wrong.

But I grabbed sdf-viewer-go and sdf-viewer and compiled the example go file and built sdf-viewer with rust. Used the command line:
```
cargo build --release --target x86_64-unknown-linux-gnu
```
As that is what I could decode from the yaml.

Attempting to launch it with the command line:
```
../../../sdf-viewer/target/release/sdf-viewer app url goexample.wasm
```
(my cwd is the folder that has the .go + .wasm files)

I get an error:

```
2024-08-18T22:42:51.764097Z ERROR sdf_viewer::sdf::wasm::load: Failed to load SDF from URL (Bad URL: failed to parse URL: RelativeUrlWithoutBase: relative URL without a base) or file (Error while importing "wasi_snapshot_preview1"."clock_time_get": incompatible import type. Expected Function(FunctionType { params: [I32, I64, I32], results: [I32] }) but received Function(FunctionType { params: [I32, I32, I32], results: [I32] }))
```

Which I assume is the reason it won't load. It defaults to try and load the demo scene on an error which is a bit misleading (who's to know what the example go is supposed to look like?)

I'm going to guess this is some rust package version mismatch where a function signature changed for something that was runtime linked. I'm not very familiar with rust or go (more of a C++ person).

Using cargo 1.80.1

**Comment by not-magic** [comment]
So this appears to be a bug in sdf-viewer with whatever version of libraries I have. Locally changing the wasi.rs entry for 'clock_time_get' to match this argument type and recompiling sdf-viewer fixes it

**Comment by Yeicor** [comment]
Thanks for the detailed error description and the suggested fix. I fixed it and added an automated integration test to help detect errors like this one in the future. Both of your reported issues are fixed in v1.5.0.
